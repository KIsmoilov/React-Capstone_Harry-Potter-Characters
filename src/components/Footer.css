.footer {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  height: 3rem;
  width: 100%;
  background-color: rgb(186, 72, 110);
  margin-top: 2rem;
}

p {
  color: white;
  font-weight: 500;
  font-size: 1.2rem;
  text-align: center;
}

@media screen and (max-width: 768px) {
  .footer {
    position: fixed;
    bottom: 0;
  }

  p {
    font-size: 1rem;
  }
}

@media screen and (max-width: 420px) {
  p {
    font-size: 0.8rem;
  }
}