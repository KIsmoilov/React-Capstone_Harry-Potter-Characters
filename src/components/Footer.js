import React from 'react';

// stylesheet
import './Footer.css';

const Footer = () => (
  <footer className="footer">
    <p>2023 All Rights Reserved. Created by Khusniddin Ismoilov. </p>
  </footer>
);
export default Footer;
