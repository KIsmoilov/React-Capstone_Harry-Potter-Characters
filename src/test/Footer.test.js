import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Footer from '../components/Footer';

it('renders correctly', () => {
  const navbar = renderer
    .create(<Router><Footer /></Router>)
    .toJSON();
  expect(navbar).toMatchSnapshot();
});
