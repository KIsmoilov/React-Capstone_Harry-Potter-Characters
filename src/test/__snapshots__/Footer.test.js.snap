// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="footer"
>
  <p>
    2023 All Rights Reserved. Created by Khusniddin Ismoilov. 
  </p>
</footer>
`;
